crypto/x509v3/v3_addr.c

```diff
@@ -1195,7 +1203,11 @@ static int addr_validate_path_internal(X509_STORE_CTX *ctx,
     } else {
         i = 0;
         x = sk_X509_value(chain, i);
-        OPENSSL_assert(x != NULL);
+        if (!ossl_assert(x != NULL)) {
+            if (ctx != NULL)
```

Comment:
Real question here if it's appropriate to override ctx->error if this condition is met. I mean it might be more appropriate to not touch it, just return 0, just like if below does...